### PR TITLE
[CDAP-14796] Change dataprep API usage to use system service

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-14796
Build: https://builds.cask.co/browse/CDAP-UDUT204

~*note: quite sure, there will be a couple of conflict with https://github.com/cdapio/cdap/pull/11020*~

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/DataPrep.scss [hunk 1]
```diff
@@ -31,8 +31,8 @@ $toggle-btn-disabled-bg-color: #cccccc;
     .dataprep-body.row {
       // height calculations
       // 100vh - height of navbar - top panel - footer
-      // 100vh - 50px - 50px - 54px
-      height: calc(100vh - 154px);
+      // 100vh - 50px - 548px - 54px
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/DataPrep/DataPrep.scss]: This is fixed in the styling PR i have open in a more generic way. https://github.com/cdapio/cdap/pull/11020/files#diff-f986deff3a889570757d014a947fc2f9R37 Can you skip this change for now?

File: cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/KafkaBrowser/KafkaBrowser.scss [hunk 2]
```diff
@@ -21,18 +21,22 @@ $header-border: #cccccc;
 $header-bg: #eeeeee;
 $content-font-color: #333333;
 $hr_color: #999999;
+$dataprep-top-panel-height: 50px;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/KafkaBrowser/KafkaBrowser.scss]: nit: we already have declared in the `DataprepBrowserVariables.scss` right? Any specific reason we have redeclared it here?

File: cdap-ui/app/cdap/components/DataPrepConnections/DatabaseConnection/DatabaseDetail.js [hunk 3]
```diff
@@ -82,7 +82,9 @@ export default class DatabaseDetail extends Component {
       properties: this.constructProperties(),
     };
 
-    MyDataPrepApi.getDatabaseList({ namespace }, requestBody).subscribe(
+    console.log('request', requestBody);
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/DataPrepConnections/DatabaseConnection/DatabaseDetail.js]: nit: remove?

File: cdap-ui/app/cdap/styles/common.scss [hunk 4]
```diff
@@ -43,9 +43,9 @@ body {
   font-family: var(--font-family);
 
   #app-container {
-    margin-top: 50px;
+    margin-top: 48px;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/styles/common.scss]: Same as the other comment: https://github.com/cdapio/cdap/pull/11020/files#diff-7aed8587c8ae4dfd5468897ea990f6baR46 already done in the style PR. Can you revert this change so that we can pick it up from there?

**Review by ajainarayanan — COMMENTED** [review]
@elfenheart Few minor comments. Otherwise 👍 LGTM

File: cdap-ui/app/cdap/components/DataPrep/DataPrep.scss (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/DataPrep/DataPrep.scss]: Yeah, I will rebase this branch with your PR once yours is merged

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM
